test/cursor.js

```diff
@@ -231,6 +231,21 @@ describe('cursor', function() {
     });
   });
 
+  it('should pass multiple arguments to operations', async () => {
+    const cursor = await db.a.findAsCursor()
+      .limit(1)
+      .addCursorFlag('noCursorTimeout', true);
```

Comment:
oh that's neat! how do we want to use this flag?

Reply:
During long-running backfills (the sfdc replica is a particular offender), sometimes the cursor will timeout mid-query since we have so many records to iterate over, or the query takes a long time to run. So, in the past I've had to manually edit node modules to set this flag on migration instances.